docker/server/start-server.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+set -o pipefail # if any code doesn't return 0, exit the script
+
+
+function setup_server() {
+	python manage.py makemigrations
+	python manage.py migrate
+}
+
+function start_server() {
+  python manage.py runserver 0.0.0.0:8000
```

Comment:
*task* Python has some excellent HTTP servers and, ideally, that's what we use in Production. Let's research what these HTTP servers are and discuss the pros and cons of using them in our development environment.

Reply:
@nbyodai To answer your question,`NGINX` can be used to serve the static files in production and `gunicorn` as  the application server and it is ideal for production environment.

Reply:
Awesome, that's what I was expecting. In my view, and to prevent a lot of headache down the line, it is best to try as much as possible to get your development, staging and prod environments to be as close as possible.

The ability to do this is one of the main selling points of using Docker in development. Let's set up to use `gunicorn` as our application server.

Reply:
@nbyodai Thanks, I totally agree with your view as well.  I think there is an environment variable that differentiates between  `development`  and `production` environment. What can be done is that at runtime the `start-server.sh` can have a condition that checks the environment and in turns determines what server should be run i.e. production or development server.

Reply:
I can update the start scripts for that scenario.